app/components/settings/categories/GeneralSettings.scss

```diff
@@ -0,0 +1,25 @@
+@import '../../../themes/mixins/error-message';
+
+.component {
+  margin-bottom: 20px;
+  :global {
+    h1 {
+      color: #5e6066;
+      font-family: $font-medium;
+      font-size: 19px;
+      line-height: 1.21;
+    }
+    p {
+      color: #5e6066;
+      font-family: $font-regular;
+      font-size: 16px;
+      line-height: 22px;
+      margin-top: 1em;
+    }
+  }
```

Comment:
Why are the `h1` and `p` styles used here? I guess those are copied over from the old settings screens? At the moment we don't have any text and it's unclear if we will need something like that.